docs/how-to/stream_ordered_allocator.rst

```diff
@@ -24,7 +24,17 @@ Disadvantages of SOMA:
 Using SOMA
```

Comment:
Here's my comment for Line 6.
The topic should be titled:
Stream-Ordered Memory Allocator
This is a compound modifier. 
It should be changed in other sections too